File: BaekJoon/2577_숫자의 개수.java
import java.util.Scanner;
public class Main {
	public static void main(String[] args) {		
		Scanner scan = new Scanner(System.in);		
		int a = scan.nextInt();
		int b = scan.nextInt();
		int c = scan.nextInt();		
		int d = a*b*c;
		int[] array = new int[10];			
		while(d > 0) { 
			int i = d%10; 
			array[i]++; 
			d /= 10; 
		}		
		for(int i : array) {
			System.out.println(i);
		}
	}
}

File: BaekJoon/2577_숫자의 개수.kt
import java.util.*
fun main(args: Array<String>) {
    var sc:Scanner = Scanner(System.`in`)
    var sum = 1
    for (i in 1..3) {
        sum *= sc.nextInt()
    }
    var saveStr = sum.toString()
    var arr = Array(10,{0})
    for (i in 0 until saveStr.length) {
        arr[saveStr.get(i).toString().toInt()]++
    }
    for (i in 0 until arr.size-1) {
        println(arr[i])
    }
    print(arr[9])
}
